sql/moz-fx-data-shared-prod/mozilla_vpn_derived/events_daily_v1/query.sql

```diff
@@ -29,6 +29,36 @@ WITH sample AS (
     mozillavpn.main e
   CROSS JOIN
     UNNEST(e.events) AS event
+  UNION ALL
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    name AS event,
+    category,
+    extra,
+    sample_id,
+    timestamp,
+    metadata,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    client_info.client_id AS client_id,
+    client_info.app_build AS app_build,
+    client_info.app_channel AS app_channel,
+    client_info.app_display_version AS app_display_version,
+    client_info.architecture AS architecture,
+    client_info.first_run_date AS first_run_date,
+    client_info.telemetry_sdk_build AS telemetry_sdk_build,
+    client_info.locale AS locale,
+    (
+      SELECT
+        ARRAY_AGG(STRUCT(key, value.branch AS value))
+      FROM
+        UNNEST(ping_info.experiments)
+    ) AS experiments
+  FROM
+    org_mozilla_firefox_vpn.main e
+  CROSS JOIN
+    UNNEST(e.events) AS event
```

Comment:
Do these two tables `events_daily_v1` and `event_type_history_v1` need to get backfilled?

Reply:
Yep!  From the deployment plan in the description:
> 2. Re-run `bqetl_event_rollup.mozilla_vpn_derived__event_types_history__v1` Airflow tasks from 2022-06-22 to date (2022-06-22 is when the earliest separate VPN Android app Glean data arrived).

> 4. Re-run `bqetl_event_rollup.mozilla_vpn_derived__events_daily__v1` Airflow tasks from 2022-06-22 to date.